plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -440,6 +440,19 @@ public class LibvirtComputingResource extends ServerResourceBase implements Serv
 
     protected Boolean enableManuallySettingCpuTopologyOnKvmVm = AgentPropertiesFileHandler.getPropertyValue(AgentProperties.ENABLE_MANUALLY_SETTING_CPU_TOPOLOGY_ON_KVM_VM);
 
+    /**
+     * Virsh command to set the memory balloon stats period.<br><br>
+     * 1st parameter: the VM name;<br>
+     * 2nd parameter: the period (in seconds).
+     */
+    private static final String COMMAND_SET_MEM_BALLOON_STATS_PERIOD = "virsh dommemstat %s --period %s --live";
```

Comment:
Why use `virsh` commands "by hand" if there is already Java bind for it via [Libvirt-Java Domain.memoryStats(int number)](https://gitlab.com/libvirt/libvirt-java/-/blob/master/src/main/java/org/libvirt/Domain.java#L1509)?
Is there something that the `memoryStats` does not support?

Reply:
Can you answer this one @joseflauzino , did you investigate the java binding?

Reply:
ping @joseflauzino

Reply:
@DaanHoogland and @GutoVeronezi 
I answered this one in [this other comment](https://github.com/apache/cloudstack/pull/6358#issuecomment-1212112323). That's why I marked it as resolved.